coreos-installer

```diff
@@ -613,9 +613,23 @@ log() {
 #########################################################
 write_image_to_disk() {
     log "Writing disk image"
-    
+
+    set -o pipefail
```

Comment:
We don't use any pipes here, so shouldn't have to toggle this on and off, right? (But ideally it'd always be on, that's https://github.com/coreos/coreos-installer/issues/42).

Reply:
Yep, I was trying to make things consistent with the rest of the code. Thanks for the link!